packages/snowflake/src/lib/Snowflake.ts

```diff
@@ -108,15 +123,15 @@ export interface SnowflakeGenerateOptions {
 
 	/**
 	 * The worker ID to use
-	 * @default 1n
+	 * @default BigInt(worker_threads.threadId & 0b11111)
```

Comment:
This is a lie. You don't `& 0b11111` here but in the generate function

Reply:
That's intentional, the OP's done later.

Reply:
Do you think people will even know what that means from this obscure default or even give a damn? The default is `BigInt(worker_threads.threadId)` in code, and it should be documented as such too

Reply:
Done, also clarified the truncation behavior.